content/projects/gsoc/2023/project-ideas/add-probes-to-plugin-health-score.adoc

```diff
@@ -53,20 +53,20 @@ video::fM2SMbppRxw[youtube,width=800,height=420,start=328]
 
 === Some simple probe ideas
 
-* comparing the existing parent pom to the latest one?
-* comparing the base Jenkins version to the latest one?
-* see if spotbugs has been configured to catch most of the bugs
-* has a contributing guide
-* has enabled incremental builds
-* has replace Nonnull and CHeckForNull annotations
+* Comparing the existing parent pom to the latest one?
+* Comparing the base Jenkins version to the latest one?
+* See if spotbugs has been configured to catch most of the bugs
+* Has a contributing guide
+* Has enabled incremental builds
+* Has replaced Nonnull and CHeckForNull annotations
 
-Existing probes are listed at link:https://plugin-health.jenkins.io/probes[Plugin Health Scoring :: Probes]
+Existing probes are listed at link:https://plugin-health.jenkins.io/probes[Plugin Health Scoring +::+ Probes]
```

Comment:
This is how I see this link in the deployed jenkins.io:
![image](https://user-images.githubusercontent.com/43273420/218133307-d928282b-ad43-432b-b31b-392151b2d1c0.png)
We'd need to fix how the link gets displayed, plus shift it to the left side so it doesn't look like a part of a bullet list (if that's the goal)

And I'm sorry, I don't know AsciiDoc well enough to suggest an edit here.